README.md

```diff
@@ -64,10 +64,23 @@ compile: 'com.github.invictum:serenity-reportportal-integration:1.5.7'
 
 At this point setup of integration is done. The only thing you have to do is to configure Report Portal itself. In general it means just adding of `reportportal.properties` file to you project tests root. Minimal properties example is described below:
 ```
+### If you are using the latest Reportportal instance then configure API KEY as below:
```

Comment:
it would be great to see exact version here, e.g. "## If you are using Reportportal instance vX.Y+ then configure API KEY as below:".
P.S. I am not sure if it is easy to find exact version when RP updated the key (I wasn't able to find it). If you are aware, please put it here, otherwise put the latest version you used for testing.
Thanks!

Reply:
Sure @grey-rain, I have tested this PR on the below version of Reportportal:

- Current version:
- API Service: 5.9.0-20;
- Index Service: 5.8.0-RC-12;
- Jobs Service: 5.8.1-24;
- Authorization Service: 5.8.1-18;
- Service UI: 5.9.0;

And additionally I noticed that with the latest java-client of Reportportal we can also pass UDID in api.key property so it seems it's backward compatible.

Reply:
so it seems like we do not need to change readme?  🤔

Reply:
Key name will still change for latest RP java-client from rp.uuid to rp.api.key

What I observed is, it works fine if we use RP v5.8.0 and pass uuid as the value.
Ex.
rp.api.key= <YOUR UUID>